packages/shared/src/features/boost/CampaignListView.tsx

```diff
@@ -0,0 +1,127 @@
+import type { ReactElement } from 'react';
+import React, { useMemo } from 'react';
+import { startOfDay } from 'date-fns';
+import type { differenceInDays } from 'date-fns';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../components/typography/Typography';
+import type { PostCampaign } from '../../hooks/post/usePostBoost';
+import { Image } from '../../components/image/Image';
+import {
+  Button,
+  ButtonColor,
+  ButtonVariant,
+} from '../../components/buttons/Button';
+import { CoreIcon, LinkIcon } from '../../components/icons';
+import { IconSize } from '../../components/Icon';
+import { DataTile } from './DataTile';
+import { BeforeIcon } from '../../components/icons/Before';
+import { ProgressBar } from '../../components/fields/ProgressBar';
+
+interface CampaignListViewProps {
+  campaign: PostCampaign;
+}
+
+export const getAbsoluteDifferenceInDays: typeof differenceInDays = (
```

Comment:
Isn't this the function you want above already?

Reply:
also actualy think date-fns has this built in, not sure we use in on apps already (do think so)

Reply:
Yeah, there is one with date-fns, but I remember it was not inline with our requirements in the past. For example, if you get the difference and it is less than 48 hours, the result is 1 day and we want two days.